src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -42,32 +45,123 @@
 public final class RqMultipartTest {
 
     /**
-     * RqMultipart can parse body.
-     * @throws IOException If some problem inside
+     * Carriage return constant.
+     */
+    private static final String CR = "\r\n";
+    /**
+     * RqMultipart can return true on equals call.
+     * @throws IOException if some problem inside
      */
     @Test
-    public void parsesHttpBody() throws IOException {
+    public void returnsTrueOnEqualsCall() throws IOException {
+        final Request req = create(
+            Joiner.on(CR).join(
+                "Content-Disposition: form-data; name=\"addres\"",
+                "",
+                "449 N Wolfe Rd, Sunnyvale, CA 94085"
+            ),
+                // @checkstyle LineLength (1 line)
+                "Content-Disposition: form-data; name=\"data\"; filename=\"a.bin\""
```

Comment:
@ekondrashev let's use new joiner here on `; ` character and avoid line length checkstyle suppression